pom.xml

```diff
@@ -117,6 +131,18 @@
                     <version>3.0.0</version>
                 </plugin>
 
+                <plugin>
+                    <groupId>org.jacoco</groupId>
+                    <artifactId>jacoco-maven-plugin</artifactId>
+                    <version>${jacoco.version}</version>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.sonarsource.scanner.maven</groupId>
+                    <artifactId>sonar-maven-plugin</artifactId>
+                    <version>${sonar-maven-plugin-version}</version>
+                </plugin>
+
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>3.0.0-M5</version>
```

Comment:
Incosinsten version - M5? When pluginManagement defines a plugin version is not required in plugins section

Reply:
Unfortunetly throw error with the new M7 version

Reply:
[8359bb6](https://github.com/BlackBeltTechnology/osgi-asciidoc-converter/pull/4/commits/8359bb6bcd7ba075217aecf5d314ebc6669de4d1)